templates/vue-demo-store/components/account/AccountLoginForm.vue

```diff
@@ -125,7 +126,7 @@ useFocus(emailImputElement, { initialValue: true });
           <slot name="action">
             <div @click="$emit('close')">
               <NuxtLink
-                to="/register"
+                :to="localePath('/register')"
```

Comment:
Maybe there's a way to overwrite NuxtLink and add this logic there? Would provide much better DX

Reply:
We can use navigation guards, but I feel that this will make the code less readable. In this case, the developer has more flexibility